acceptance-tests/tests/src/test/resources/jsonrpc/eth/getBlockBy/test-cases/02_execute_payload_1.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Why does the first test start from 02?

Reply:
haha, I knew you'd pick up on that ;)

I copied the test cases over from replayBlock ATs and I was too lazy to rename all the files after deleting the unnecessary first 01_prepare_payload.
I will rename :p